docs/DeveloperGuide.md

```diff
@@ -36,15 +36,34 @@ Step 2. The user executes `add stock; i/tomato; q/10; p/$0.40;` command to add a
 Step 3. The user can now search against the current `stock` to see if an ingredient is stored in the `stock`. The user now executes `search stock; k/tomato;`, which will display the following result in the image. 
 
 <p align="center">
-  <img src="https://user-images.githubusercontent.com/59989652/77921374-47b58b00-72d2-11ea-84bd-fe0c790b1d59.PNG">
+  <img src="https://user-images.githubusercontent.com/59989652/78652378-3da81380-78f4-11ea-8b7a-63115ead3ca5.PNG">
+</p>
+
+Step 4. If the ingredient that the user is searching for does not exist within the stock, a different message will be displayed as shown in the following image.
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/59989652/78652553-75af5680-78f4-11ea-9df1-5aa85951d216.PNG">
 </p>
 
 The following sequence diagram shows how the search operation works:
 
 <p align="center">
-  <img src="https://user-images.githubusercontent.com/59989652/77916616-e8ed1300-72cb-11ea-9ba0-79e1ccc119ab.png">
+  <img src="https://user-images.githubusercontent.com/59989652/78683492-efaa0480-7921-11ea-82c4-e2b4abb9da92.png">
 </p>
 
+
+The sequence diagram can be interpreted as such:
+* 1. `CommandParser` calls its own `CommandParser#parseCommand(...)`. 
+* 2. Assuming the user input the search stock command correctly, `SearchStockCommand#SearchStockCommand(...)` constructor is called. 
+* 3. The newly constructed `SearchStockCommand` invokes its `SearchStockCommand#parseIntoSearchKeyword(...)` and does not return anything.
+* 4. The `CommandParser` then invokes `SearchStockCommand#execute(...)`, which then further invokes `Stock#searchStock(...)` from the Stock object.
+* 5. The `Stock` object then self-invoke `Stock#checkIngredientInStock(...)` to see if there are ingredients that matches keyword that was passed into earlier.
+* 6. If there are search results, `Stock#printSearchResult(...)` will display all the ingredients that matches the keyword given.
+* 7. If there is no ingredient that matches the keyword, the program will display a different message to show the user.
+* 8. Next, the time line returns back to `CommandParser` and the `SearchStockCommand` object is destroyed here.
+* 9. If however, the user input did not input the search stock command correctly, `CommandParser` will invoke `CommandParser#errorCommand()` to notify the user.
```

Comment:
![image](https://user-images.githubusercontent.com/49228945/78783287-f5f9b880-79d5-11ea-9fa9-37c830e3c4b9.png)
The format seems problematic when viewing on github